auto-deployment broken

The release 1.2.40 failed partially.
Deployment towards Maven Central repository worked fine.
But the deployment plugin for `release-monthly.yaml` failed because of:

```
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  17:36 min
[INFO] Finished at: 2026-06-17T19:47:41Z
[INFO] ------------------------------------------------------------------------
Error:  Failed to execute goal org.sonatype.central:central-publishing-maven-plugin:0.5.0:publish (injected-central-publishing) on project fast-isbn: Execution injected-central-publishing of goal org.sonatype.central:central-publishing-maven-plugin:0.5.0:publish failed: com.fasterxml.jackson.databind.exc.UnrecognizedPropertyException: Unrecognized field "warnings" (class org.sonatype.central.publisher.client.model.DeploymentApiResponse), not marked as ignorable (6 known properties: "deploymentId", "purls", "cherryBomUrl", "errors", "deploymentState", "deploymentName"])
Error:   at [Source: REDACTED (`StreamReadFeature.INCLUDE_SOURCE_IN_LOCATION` disabled); line: 1, column: 209] (through reference chain: org.sonatype.central.publisher.client.model.DeploymentApiResponse["warnings"])
Error:  -> [Help 1]
Error:  
Error:  To see the full stack trace of the errors, re-run Maven with the -e switch.
Error:  Re-run Maven using the -X switch to enable full debug logging.
Error:  
Error:  For more information about the errors and possible solutions, please read the following articles:
Error:  [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/PluginExecutionException
Error: Process completed with exit code 1.
```

--- a/README.md
+++ b/README.md
@@ -6,8 +6,8 @@ This library can evaluate **millions of ISBNs per second** on a single CPU core.
 [![ISBN-13](https://creativecouple.de/validation/isbn/9789980431929.banner.svg)](https://creativecouple.de/validation/isbn/#?9789980431929)
 [![Java Version](https://img.shields.io/badge/Java-1.8%2B-orange)](https://www.oracle.com/java/technologies/javase/javase-jdk8-downloads.html)
 [![License](https://img.shields.io/badge/License-MIT-blue.svg)](https://opensource.org/licenses/MIT)
-[![Download](https://img.shields.io/maven-central/v/net.creativecouple.validation/fast-isbn?version=1.2.39)](https://central.sonatype.com/artifact/net.creativecouple.validation/fast-isbn)
-[![Javadocs](https://javadoc.io/badge2/net.creativecouple.validation/fast-isbn/javadoc.svg?version=1.2.39)](https://javadoc.io/doc/net.creativecouple.validation/fast-isbn)
+[![Download](https://img.shields.io/maven-central/v/net.creativecouple.validation/fast-isbn?version=1.2.40)](https://central.sonatype.com/artifact/net.creativecouple.validation/fast-isbn)
+[![Javadocs](https://javadoc.io/badge2/net.creativecouple.validation/fast-isbn/javadoc.svg?version=1.2.40)](https://javadoc.io/doc/net.creativecouple.validation/fast-isbn)
 
 # fast-isbn – world's fastest ISBN range validation
 A common issue in digital commerce systems is to validate article identifiers.
@@ -46,7 +46,7 @@ It normalizes and corrects the given string input with respect to hyphens/dashes
 parsing both ISBN-10 and ISBN-13 numbers into the same object, the
 [`ISBN`](https://javadoc.io/doc/net.creativecouple.validation/fast-isbn/latest/de/creativecouple/validation/isbn/ISBN.html). 
 
-## Usage (latest version "1.2.39")
+## Usage (latest version "1.2.40")
 
 You only need to install [`net.creativecouple.validation:fast-isbn`](https://mvnrepository.com/artifact/net.creativecouple.validation/fast-isbn/latest)
 as dependency in your Java/Kotlin/Scala project definition,
@@ -60,7 +60,7 @@ as dependency in your Java/Kotlin/Scala project definition,
     <dependency>
         <groupId>net.creativecouple.validation</groupId>
         <artifactId>fast-isbn</artifactId>
-        <version>1.2.39</version>
+        <version>1.2.40</version>
     </dependency>
 </dependencies>
 ```
@@ -69,14 +69,14 @@ as dependency in your Java/Kotlin/Scala project definition,
 <summary>as Gradle dependency in your <code>build.gradle</code>,</summary>
 
 ```gradle
-implementation group: 'net.creativecouple.validation', name: 'fast-isbn', version: '1.2.39'
+implementation group: 'net.creativecouple.validation', name: 'fast-isbn', version: '1.2.40'
 ```
 </details>
 <details>
 <summary>or as Scala dependency in your <code>build.sbt</code>.</summary>
 
 ```scala
-libraryDependencies += "net.creativecouple.validation" % "fast-isbn" % "1.2.39"
+libraryDependencies += "net.creativecouple.validation" % "fast-isbn" % "1.2.40"
 ```
 </details>
 

--- a/pom.xml
+++ b/pom.xml
@@ -49,31 +49,31 @@
         <dependency>
             <groupId>org.assertj</groupId>
             <artifactId>assertj-core</artifactId>
-            <version>3.24.2</version>
+            <version>3.27.7</version>
             <scope>test</scope>
         </dependency>
         <dependency>
             <groupId>org.junit.jupiter</groupId>
             <artifactId>junit-jupiter-api</artifactId>
-            <version>5.10.0</version>
+            <version>6.1.0</version>
             <scope>test</scope>
         </dependency>
         <dependency>
             <groupId>org.junit.jupiter</groupId>
             <artifactId>junit-jupiter-engine</artifactId>
-            <version>5.10.0</version>
+            <version>6.1.0</version>
             <scope>test</scope>
         </dependency>
         <dependency>
             <groupId>org.junit.jupiter</groupId>
             <artifactId>junit-jupiter-params</artifactId>
-            <version>5.10.0</version>
+            <version>6.1.0</version>
             <scope>test</scope>
         </dependency>
         <dependency>
             <groupId>org.junit.platform</groupId>
             <artifactId>junit-platform-launcher</artifactId>
-            <version>1.10.0</version>
+            <version>6.1.0</version>
             <scope>test</scope>
         </dependency>
     </dependencies>
@@ -89,7 +89,7 @@
             <plugin>
                 <groupId>org.sonatype.central</groupId>
                 <artifactId>central-publishing-maven-plugin</artifactId>
-                <version>0.5.0</version>
+                <version>0.11.0</version>
                 <extensions>true</extensions>
                 <configuration>
                     <publishingServerId>central</publishingServerId>
@@ -100,7 +100,7 @@
             <plugin>
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-source-plugin</artifactId>
-                <version>3.3.1</version>
+                <version>3.4.0</version>
                 <executions>
                     <execution>
                         <id>attach-sources</id>
@@ -113,7 +113,7 @@
             <plugin>
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-javadoc-plugin</artifactId>
-                <version>3.6.3</version>
+                <version>3.12.0</version>
                 <executions>
                     <execution>
                         <id>attach-javadocs</id>
@@ -126,7 +126,7 @@
             <plugin>
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-gpg-plugin</artifactId>
-                <version>3.1.0</version>
+                <version>3.2.8</version>
                 <executions>
                     <execution>
                         <id>sign-artifacts</id>
@@ -146,7 +146,7 @@
             <plugin>
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-surefire-plugin</artifactId>
-                <version>3.1.2</version>
+                <version>3.5.6</version>
             </plugin>
         </plugins>
     </build>

--- a/pom.xml
+++ b/pom.xml
@@ -6,7 +6,7 @@
 
     <groupId>net.creativecouple.validation</groupId>
     <artifactId>fast-isbn</artifactId>
-    <version>1.2.39</version>
+    <version>1.2.40</version>
     <packaging>jar</packaging>
 
     <name>Fast ISBN-Range Validation</name>

